packages/react-data-grid/src/cells/headerCells/SortableHeaderCell.js

```diff
@@ -52,13 +53,13 @@ class SortableHeaderCell extends React.Component {
       'react-grid-HeaderCell-sortable--ascending': this.props.sortDirection === 'ASC',
       'react-grid-HeaderCell-sortable--descending': this.props.sortDirection === 'DESC'
     });
-
+    let content = this.props.headerRenderer ? this.props.headerRenderer :  this.props.column.name;
     return (
       <div className={className}
         onClick={this.onClick}
         style={{cursor: 'pointer'}}>
+        {content}
```

Comment:
Is changing the order intentional?

Reply:
This is an old PR. It seems that the order has been changed in master since it was created. I'll keep it to what it is in master